pkg/ui/src/util/locations.ts

```diff
@@ -89,3 +89,44 @@ export function findOrCalculateLocation(locations: LocationTree, locality: Local
   centroid = vector.mult(centroid, 1 / childLocations.length);
   return { longitude: centroid[0], latitude: centroid[1] };
 }
+
+/*
+ * getConfigStats finds how many nodes are missing locality or location config.
+ */
+export function getConfigStatus(
```

Comment:
can you add tests for this method please?

Reply:
Removed this method.